tests/bluetooth/bsim_bt/compile_permutate_kconfigs.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright 2018 Oticon A/S
+# SPDX-License-Identifier: Apache-2.0
+
+# Compile all the applications needed by the bsim_bt tests
+
+#set -x #uncomment this line for debugging
+
+: "${BSIM_OUT_PATH:?BSIM_OUT_PATH must be defined}"
+: "${BSIM_COMPONENTS_PATH:?BSIM_COMPONENTS_PATH must be defined}"
+: "${ZEPHYR_BASE:?ZEPHYR_BASE must be set to point to the zephyr root\
+ directory}"
+
+WORK_DIR="${WORK_DIR:-${ZEPHYR_BASE}/bsim_bt_out}"
+BOARD="${BOARD:-nrf52_bsim}"
+BOARD_ROOT="${BOARD_ROOT:-${ZEPHYR_BASE}}"
+
+mkdir -p ${WORK_DIR}
+
+source ${ZEPHYR_BASE}/tests/bluetooth/bsim_bt/compile.source
+
+declare -a list=(
+#"CONFIG_BT_DATA_LEN_UPDATE=n"
+#"CONFIG_BT_PHY_UPDATE=n"
```

Comment:
wouldn't it be better to explicitly set a config to 'y' or 'n' instead of assuming that the default is 'y'?

Reply:
Updated the script accordingly